modular_skyrat/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -91,4 +91,50 @@
 	glass_icon_state = "synthignonglass"
 	glass_name = "Glass of Synthignon"
 	glass_desc = "Someone mixed good wine and robot booze. Romantic, but atrocious."
-	taste_description = "fancy motor oil"
\ No newline at end of file
+	taste_description = "fancy motor oil"
+
+/datum/reagent/consumable/ethanol/balikitwine
+	name = "Balikit Wine"
+	description = "If you ever wanted to know what being bitten was like..."
+	color = "#664300" // rgb: 102, 67, 0
+	nutriment_factor = 0
+	boozepwr = 60 
+	quality = DRINK_FANTASTIC // 4 out of 6
+	metabolization_rate = REAGENTS_METABOLISM 
+	taste_description = "the need to go to medical"
+	glass_icon = 'modular_skyrat/icons/obj/drinks.dmi'
+	glass_icon_state = "bagatjiwineglass"
+	glass_name = "Balikit Wine"
+	glass_desc = "They put it in a decanter."
+	value = REAGENT_VALUE_RARE
+
+/datum/reagent/consumable/ethanol/balikitwine/on_mob_life(mob/living/carbon/C)
+	var/mob/living/carbon/human/H = C
+	if(istype(H) && H.physiology.footstep_type != FOOTSTEP_MOB_CRAWL) // Bit of a hacky way to check if the consumer is a snake taur. Only snektaurs have this though.
+		H.adjustToxLoss(3*REM) // Bagatji (and all snake taurs) take no ill effects.
```

Comment:
Everything but this is okay. It's currently 39 toxin damage per single sip, given the not super hard to obtain crafting recipe it probably should be lowered just a tad.
Lowering this to 2.5 will deal 32.5 toxin damage per sip.
Lowering this to 2 will deal 26 toxin damage per sip.

Change to either 2 or 2.5 and ping me at Floof Ball#0798 once done and once I'm able I'll merge it. I can quickly do it for you if you also ping me to say it's okay.